src/platforms/android/common/dependencies.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Managing Dependencies
```

Comment:
Since this page is devoted to using a BOM, the page should reflect that
```suggestion
title: Using a BOM
description: "Learn more about using a bill of materials with multiple Sentry dependencies."
```

Reply:
I also agree that this isn't quite the right spot. I'd do the following:
1. Move the page to a subpage of Configuration
2. Add a link to the page where we currently tell users about NDK and the dependency on Gson (`src/includes/getting-started-install/android`) adding "If you are using multiple Sentry dependencies, you can add a  [bill of materials](link to new page) to avoid specifying the version of each dependency."